### PR TITLE
Fostrich safe admin page

### PR DESCRIPTION
**Pull request by frost-ostrich**
# Description
Add a Safe admin page which acts as backup when the Safe App ui goes offline.

The preview of this admin page can be accessed in https://pr-935-preview.templedao.link/safe

# Checklist
- [ ] Code follows the style guide
- [ ] I have performed a self-review of my own code
- [ ] New and existing tests pass locally
- [ ] This PR is targeting the correct branch

### CODE REVIEW

**Comment by nichosystem** [comment]
Deploy preview for _core_ ready!

✅ Preview
https://core-hksoznig8-templedao.vercel.app
https://pr-935-preview.templedao.link

Built with commit d8aecc45b66e69c3fb3a4573df7ddae86c0b7c5b.
This pull request is being automatically deployed with [vercel-action](https://github.com/marketplace/actions/vercel-action)

File: apps/dapp/src/constants/env/production.tsx
Comment (frost-ostrich) [inline apps/dapp/src/constants/env/production.tsx:160]: need the mainnet msig addresses from temple

File: apps/dapp/src/safe/sdk/contracts/config.ts
Comment (mountainpath9) [inline apps/dapp/src/safe/sdk/contracts/config.ts:2]: Given this code is sourced from the official safe sdk, can you comment with a link to the original source?

File: apps/dapp/src/safe/sdk/contracts/contractInstancesEthers.ts
Comment (mountainpath9) [inline apps/dapp/src/safe/sdk/contracts/contractInstancesEthers.ts:8]: ditto on link to original source.

File: apps/dapp/src/safe/sdk/contracts/safeDeploymentContracts.ts
Comment (mountainpath9) [inline apps/dapp/src/safe/sdk/contracts/safeDeploymentContracts.ts:24]: ditto on link to original source.

File: apps/dapp/src/safe/sdk/signatures/SafeSignature.ts
Comment (mountainpath9) [inline apps/dapp/src/safe/sdk/signatures/SafeSignature.ts:2]: ditto on link to original source.

File: apps/dapp/src/safe/sdk/signatures/utils.ts
Comment (mountainpath9) [inline apps/dapp/src/safe/sdk/signatures/utils.ts:5]: ditto on link to original source.

File: apps/dapp/src/safe/sdk/transactions/gas.ts
Comment (mountainpath9) [inline apps/dapp/src/safe/sdk/transactions/gas.ts:5]: ditto on link to original source.

File: apps/dapp/src/safe/sdk/use-safe-sdk.tsx
Comment (mountainpath9) [inline apps/dapp/src/safe/sdk/use-safe-sdk.tsx:24]: ditto on link to original source.

**Comment by mountainpath9** [comment]
Actually, rather than adding comments in every file in `safe/sdk`, perhaps a README file that that provides a list of the files from which the code was sourced?

**Comment by mountainpath9** [comment]
This needs a review from one of the core devs more familar with the temple dapp codebase.

File: apps/dapp/src/components/Pages/Safe/admin/SafeTxDataTable.tsx
Comment (frost-ostrich) [reply apps/dapp/src/components/Pages/Safe/admin/SafeTxDataTable.tsx:34]: don't think it's necessary since that the default value of the state?

File: protocol/package.json
Comment (nichosystem) [reply protocol/package.json:47]: I don't have a strong preference. /protocol/scripts works, but it could also be appropriate in apps/dapp if the generated typechain would exclusively be used in the dapp

Comment (frost-ostrich) [reply protocol/package.json]: moved to dapp

**Review by frontier159 — APPROVED** [review]

**Pull request merged**
